tests/core/test_asyncify_assertions.c

```diff
@@ -30,4 +30,5 @@ int main() {
   suspend();
   suspend();
   printf("finish\n");
+  return 0;
```

Comment:
is this needed?

Reply:
No, its generally a change I make to any test I see that doesn't have it.    Its an unrelated test here though so I will revert.